#!/usr/bin/env python
PACKAGE = "franka_advanced_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("joint_1", double_t, 0, "Stiffness_joint_1", 10, 0, 600)
gen.add("joint_2", double_t, 0, "Stiffness_joint_2", 10, 0, 600)
gen.add("joint_3", double_t, 0, "Stiffness_joint_3", 10, 0, 600)
gen.add("joint_4", double_t, 0, "Stiffness_joint_4", 10, 0, 600)
gen.add("joint_5", double_t, 0, "Stiffness_joint_5", 10, 0, 250)
gen.add("joint_6", double_t, 0, "Stiffness_joint_6", 10, 0, 150)
gen.add("joint_7", double_t, 0, "Stiffness_joint_7", 5, 0, 50)
gen.add("damping_ratio", double_t, 0, "Damping_ratio", 0.77, 0, 2)

exit(gen.generate(PACKAGE, "dynamic_compliance", "compliance_joint_param"))
